pkg/vm/evm_prompts.go

```diff
@@ -200,7 +200,7 @@ func PromptSubnetEVMGenesisParams(
 		}
 	}
 
-	// Gas Kind
+	// Gas Type
```

Comment:
you probably want to rename other Gas Kind relayer stuff, eg promptGasTokenKind, defaultsKind, etc. I mean, this comment relates to Gas Kind so all other stuff also applies (or keep comment unmodified for the moment on)